Update playbooks_reuse_roles.rst (#1275)
ansible/ansible-documentation: docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

Role directory structure
========================

> An Ansible role has a defined directory structure with six main standard directories. You must include at least one of these directories in each role. You can omit any directories the role does not use. For example:

Author: seven

.. include:: shared_snippets/role_directory.txt

By default, Ansible will look in most role directories for a ``main.yml`` file for relevant content (also ``main.yaml`` and ``main``):

- ``tasks/main.yml`` - A list of tasks that the role provides to the play for execution.

> - ``handlers/main.yml`` - handlers, which are imported into the parent play for use by the role or other roles and tasks in the play.

Reviewer: ```suggestion
- ``handlers/main.yml`` - handlers that are imported into the parent play for use by the role or other roles and tasks in the play.
```

- ``vars/main.yml`` - high precedence variables provided by the role to the play (see :ref:`playbooks_variables` for more information).
- ``files/stuff.txt`` - one or more files that are available for the role and it's children.
- ``templates/something.j2`` - templates to use in the role or child roles.
- ``meta/main.yml`` - metadata for the role, including role dependencies and optional Galaxy metadata such as platforms supported. This is required for uploading into galaxy as a standalone role, but not for using the role in your play.

.. note::
   - None of the files above are required for a role. For example, you can just provide ``files/something.txt`` or ``vars/for_import.yml`` and it will still be a valid role.
   - On stand alone roles you can also include custom modules and/or plugins, for example ``library/my_module.py``, which may be used within this role (see :ref:`embedding_modules_and_plugins_in_roles` for more information).

>    - A 'stand alone' role refers to role that is not part of a collection but their own individually installable content.

Reviewer: ```suggestion
   - A 'stand alone' role refers to role that is not part of a collection but as individually installable content.
```

        name: "apache2"
        state: present

> Or call them directly when loading the role (this bypasses the main.yml files):

Reviewer: ```suggestion
Or call those tasks directly when loading the role, which bypasses the ``main.yml`` files:
```

Using roles
===========

> You can use roles in four ways:

Reviewer: ```suggestion
You can use roles in the following ways:
```
